dockerfiles/startup.sh

```diff
@@ -2,4 +2,5 @@
 
 trap /home/centos/streamwebs/dockerfiles/cleanup.sh EXIT
 bower install --allow-root
+yes "yes" | python /home/centos/streamwebs/streamwebs_frontend/manage.py migrate
```

Comment:
Alright, I understand now. We should make note that migrations files are needed for changes to schema in PRs now.